index.bs

```diff
@@ -28,6 +28,12 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     urlPrefix: dom.html
         type: interface
             text: Document; url: document
+    urlPrefix: infrastructure.html
```

Comment:
All of these should link automatically without having to be added here. Are they not?